tests/main/media-sharing/task.yaml

```diff
@@ -4,17 +4,16 @@ details: |
     the mount namespace used by snap applications into the main mount
     namespace.
 prepare: |
+    . $TESTSLIB/snaps.sh
+    install_local_devmode test-snapd-tools
     mkdir -p /media/src
     mkdir -p /media/dst
     touch /media/src/canary
-    snap install --devmode snapd-hacker-toolbelt
 execute: |
-    export PATH=$PATH:/snap/bin
     test ! -e /media/dst/canary
-    snapd-hacker-toolbelt.busybox sh -c 'mount --bind /media/src /media/dst'
+    test-snapd-tools.cmd mount --bind /media/src /media/dst
```

Comment:
@zyga indeed, we need `test-snapd-tools` in the store at least for the auto-refresh tests, probably for others too. 
Because we don't use all of its features from the store version it is out of sync from the source on snapd's tree, I'll push an up to date version to the store asap.
IMO given the low change rate of this snap we don't need to put in place a function to watch the source and have it updated in the store, but can be done easily, let me know what do you think.